doc/source/release-notes/Percona-Server-5.7.37-40.rst

```diff
@@ -0,0 +1,68 @@
+.. _PS-5.7.37-40:
+
+================================================
+*Percona Server for MySQL* 5.7.37-40
+================================================
+:Date: March 31, 2022
+
+`Percona Server for MySQL <https://www.percona.com/software/mysql-database/percona-server>`_ 5.7.37-40
+includes all the features and bug fixes available in `MySQL 5.7.37 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-37.html>`__ in addition to enterprise-grade features developed by Percona.
+
+Percona Server for MySQL is a free, fully compatible, enhanced, and open source drop-in replacement for any MySQL database. It delivers more value to MySQL server users with optimized performance, greater performance scalability and availability, enhanced backups, and increased visibility. `Commercial support contracts are available <https://www.percona.com/services/support/mysql-support>`__.
+ 
+.. contents::
+   :local:
+
+Release Highlights
+=================================================
+
+The following lists a number of the notable updates and fixes for MySQL 5.7.37, provided by Oracle, and included in Percona Server for MySQL:
+
+* The GnuPG build key has been updated. The GnuPG build key is used to sign MySQL downloadable packages. The systems configured to use repo.mysql.com may report a signature verification error when upgrading using *apt* or *yum* to MySQL 5.7.37 and higher or to MySQL 8.0.28 and higher. To resolve this issue, use one of the following methods:
+
+  * Reinstall the MySQL APT or YUM repository setup package from https://dev.mysql.com/downloads/.
+
+  * Download the MySQL GnuPG public key and add it to your system GPG keyring.
+
+  * Find MySQL APT repository instructions in `Appendix A: Adding and Configuring the MySQL APT Repository Manually <https://dev.mysql.com/doc/mysql-apt-repo-quick-guide/en/#repo-qg-apt-repo-manual-setup>`__.
+
+  * Find MySQL YUM repository instructions in `Upgrading MySQL with the MySQL Yum Repository <https://dev.mysql.com/doc/mysql-yum-repo-quick-guide/en/#repo-qg-yum-upgrading>`__. 
```

Comment:
this part can be removed as this concerns Oracle's repos and packages.

Reply:
Removed